### PR TITLE
Schedule Tidying

### PR DESCRIPTION
**Pull request by zeth**

### CODE REVIEW

File: content/abstracts.rst [hunk 1]
```diff
@@ -458,3 +474,30 @@ Monday is the Sprint day, we split into small groups and in each group
...
+Read more here: http://dont-be-afraid-to-commit.readthedocs.org
+
+.. _sciencesprint:
+
+Python for Science Sprint
+-------------------------
+
+Collaborative open science sprint where you can bring along a task to
+automate, or a piece of code you want to open source, and we will put
+you in groups to turn your existing scripts or workflow into an
```
Comment (inglesp) [inline content/abstracts.rst]: Don't think that "we will put you in groups" is the right phrasing.  How about "you will find people to help you"?

**Comment by inglesp** [comment]
One minor comment about wording, otherwise looks good to me.

**Comment by zeth** [comment]
Well that is @snim2 's original text just moved out of the schedule to save space.

**Comment by zeth** [comment]
It is a good improvement though. It does sound a bit less paternal.

**Comment by snim2** [comment]
I was aiming this at people who have never been to a sprint and might
expect some guidance. However, happy to make this change.

**Comment by inglesp** [comment]
How about "we will help you find others"?

**Comment by snim2** [comment]
Sure

**Comment by zeth** [comment]
Can someone merge this?

**Pull request merged**